### ISSUE TITLE
Showing default expanded view with timeline of active phase(s) for customer

### ISSUE THREAD

**Issue by RishiRajSahu** [appirio-tech/connect-app#2478]
Make sure the customer can see the currently active phase card. By design we wanted to expand the currently active card to display the timeline when the customer loads the dashboard and project plan (save a click to the customer).

continued from https://github.com/appirio-tech/connect-app/issues/2301

**Comment by RishiRajSahu** [comment]
@vic-topcoder there could be more than one phase active at a time so do we need to show expanded view for all the active phases with timeline tab ? it will make phase accommodate most of the UI.

**Comment by vic-topcoder** [comment]
I'd say yes. @acshields what do you think?

**Comment by acshields** [comment]
Yes, agreed.

**Comment by vikasrohit** [comment]
@RishiRajSahu unblocked this issue for next CF challenge as now we have answers for the questions.

**Comment by maxceem** [comment]
Just to confirm, only customers have to see active phases expanded, for other user roles, phases are collapsed as now, correct?

**Comment by RishiRajSahu** [comment]
@maxceem As the actual impacting entity due to this behaviour is customer only so Yes.

**Comment by maxceem** [comment]
@Gauravseta Yes there will be the next challenge in some time soon, but no exact date yet. Issues marked as "cf-pipeline" are under inner work. They may be included into the challenge or they could be also canceled, changed or postponed.

**Comment by sumitdaga** [comment]
@maxceem 
this needs to happen on both dashboard and project plan tabs or only project plan tab ?

**Comment by maxceem** [comment]
@sumitdaga this needs to happen on both tabs.

**Comment by sumitdaga** [comment]
@maxceem 
actually i think dashboard shows all phases which are active ...so all phase cards will be expanded on dashboard right?

**Comment by maxceem** [comment]
@sumitdaga correct, all phases will be opened on the dashboard tab as long as we show only active one there.

**Comment by sumitdaga** [comment]
@maxceem 
whats the best way to see if the user is a `customer` in the connect app ..will save a lot of searching if you already know this!

**Comment by maxceem** [comment]
@sumitdaga you can use user `phash_customer`/`topcoder123` to see a project as a customer.

You can create a project using manager user (`pshah_manager`/`topcoder123`) so you can activate phases using manager. And after you add customer user to the team as a member, so customer get access to the project.

After you can validate that manager see phases closed, while customer see them opened. (When you implement it).

If you are interested to know how to check in code see example https://github.com/appirio-tech/connect-app/blob/dev/src/projects/detail/containers/ProductTimelineContainer.jsx#L89

Most likely you already have `isCustomer` property in some of parent components.

**Comment by sumitdaga** [comment]
@maxceem 
ready for review!  PR #2597   
we could also create a new action/reducer `EXPAND_ALL_ACTIVE_PHASES` to avoid calling `EXPAND_PROJECT_PHASE` separately like i have done ..but it will take more work

**Comment by maxceem** [comment]
> we could also create a new action/reducer EXPAND_ALL_ACTIVE_PHASES to avoid calling EXPAND_PROJECT_PHASE separately like i have done ..but it will take more work

@sumitdaga I think it's ok for now as mostly we won't have too many active phases.
PR is merged, thanks.
